setup.rb

```diff
@@ -131,6 +131,7 @@ def system_with_debug(*args)
 system('yum clean all')
 
 system('yum -y update nss')
+system('yum -y update ca-certificates')
```

Comment:
Install would be *slightly* faster if you combined nss and ca-certificates into one update line